gis_processor/files_db_gis.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime
+from os import PathLike
+from pathlib import Path
+from typing import Union
+from uuid import UUID
+
+from acacore.database.files_db import FileDB
+from acacore.models.file import File
+
+
+class GisFilesDB(FileDB):
+    """Class for interacting with the `files.db` database, inherits form acacore `FileDB`."""
+
+    def __init__(self, database: Union[str, bytes, PathLike[str], PathLike[bytes]]) -> None:
```

Comment:
I think my problem is in here, in this class - none of the places where i call the class and use its methods seems to work adn update the files.db when the tool rearranges files.
I think maybe i acces the database and open it in the wrong way - but when i run it i just dont get any errors